### PR TITLE
update to powsybl 3.5

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

File: src/test/java/org/gridsuite/balances/adjustment/server/BalancesAdjustmentTest.java [hunk 1]
```diff
@@ -137,24 +137,24 @@ public void testSuccessBalancesAdjustmentComputation() throws InterruptedExcepti
         assertEquals(2, balanceComputationResult.getIterationCount());
 
         // BELGIUM
-        assertEquals(724.8642, testNetwork.getGenerator("BBE1AA1 _generator").getTargetP(), 0.001);
-        assertEquals(1208.1071, testNetwork.getGenerator("BBE3AA1 _generator").getTargetP(), 0.001);
-        assertEquals(1449.7285, testNetwork.getGenerator("BBE2AA1 _generator").getTargetP(), 0.001);
+        assertEquals(724.8642, testNetwork.getGenerator("BBE1AA1 _generator").getTargetP(), 0.1);
```
Comment (ChamseddineBhd) [inline src/test/java/org/gridsuite/balances/adjustment/server/BalancesAdjustmentTest.java:140]: why you have changed this value from 0.001 to 0.1 !

File: src/test/java/org/gridsuite/balances/adjustment/server/BalancesAdjustmentTest.java [hunk 2]
```diff
@@ -169,7 +169,7 @@ public void testFailedWorkingBalancesAdjustmentComputation() throws InterruptedE
         InputStream targetNetPositionsIStream = new FileInputStream(ResourceUtils.getFile("classpath:failingTargetNetPositions.json"));
         BalanceComputationResult balanceComputationResult = balancesAdjustmentService.computeBalancesAdjustment(testNetworkId, balanceComputationParameters, targetNetPositionsIStream, true, false);
         assertEquals(BalanceComputationResult.Status.FAILED, balanceComputationResult.getStatus());
-        assertEquals(11, balanceComputationResult.getIterationCount());
+        assertEquals(10, balanceComputationResult.getIterationCount());
```
Comment (ChamseddineBhd) [inline src/test/java/org/gridsuite/balances/adjustment/server/BalancesAdjustmentTest.java:172]: why balanceComputationResult.getIterationCount() returns 10 and not 11 as before ?

File: src/test/java/org/gridsuite/balances/adjustment/server/BalancesAdjustmentTest.java (hunk 1 above)
Comment (FranckLecuyer) [reply src/test/java/org/gridsuite/balances/adjustment/server/BalancesAdjustmentTest.java:140]: before, balances adjustment was using powsybl core 3.3.0 and open load flow 0.3.0
We are now in 3.5.0 and 0.5.0, and the load flow results are not exactly the same ...

File: src/test/java/org/gridsuite/balances/adjustment/server/BalancesAdjustmentTest.java (hunk 2 above)
Comment (FranckLecuyer) [reply src/test/java/org/gridsuite/balances/adjustment/server/BalancesAdjustmentTest.java:172]: before, balances adjustment was using powsybl core 3.3.0 and open load flow 0.3.0
We are now in 3.5.0 and 0.5.0, and the load flow results are not exactly the same ...

**Comment by ChamseddineBhd** [comment]
> does 3.5 modified the results of some function ? I don't understand the tests changes.

@FranckLecuyer  same as Abdel, i don't understand why iteration count has changed in the test

**Comment by jonenst** [comment]
Do we want to update to the newly released balance adjustement 1.2.1 using powsybl-core 1.5 ?

**Comment by geofjamg** [comment]
> Do we want to update to the newly released balance adjustement 1.2.1 using powsybl-core 1.5 ?
> 
> 

Yes

**Pull request merged**